src/types.rs

```diff
@@ -1555,15 +1861,52 @@ impl Rule {
         });
         lhs.chain(rhs).collect()
     }
-    /// Get a specific subterm in a Rule
+    /// Get a specific subterm in a `Rule`.
+    ///
+    /// See [`Term`] for more information.
```

Comment:
Could you link to `Term::at` to give a more specific cross-reference?

The link syntax would be ``[`Term::at`]: enum.Term.html#method.at``.